app/grandchallenge/cases/models.py

```diff
@@ -292,6 +309,24 @@ class Image(UUIDModel):
     color_space = models.CharField(
         max_length=5, blank=False, choices=COLOR_SPACES
     )
+    patient_id = models.CharField(max_length=64, null=True, blank=True)
+    patient_name = models.CharField(max_length=324, null=True, blank=True)
+    patient_birth_date = models.DateField(null=True, blank=True)
+    patient_age = models.CharField(max_length=4, null=True, blank=True)
+    patient_sex = models.CharField(
+        max_length=1,
+        null=True,
+        blank=True,
+        choices=PATIENT_SEX_CHOICES,
+        default=PATIENT_SEX_EMTPY,
+    )
+    study_date = models.DateField(null=True, blank=True)
+    study_instance_uid = models.CharField(max_length=64, null=True, blank=True)
+    series_instance_uid = models.CharField(
+        max_length=64, null=True, blank=True
+    )
+    study_description = models.CharField(max_length=64, null=True, blank=True)
+    series_description = models.CharField(max_length=64, null=True, blank=True)
```

Comment:
Ensure that these field limits are validated/truncated in PanImg

Reply:
They are validated using regex, but only in the dicom/mhd -> SimpleITKImage conversion step. Do you want me to add an extra validation step in the `SimpleITKImage` -> `PanImg` conversion step? This should not make a difference currently, but if we add the extra fields to other image builders later and forget to add the validation, an extra validation step will catch that developer error.

Reply:
Can you try it? I think it will be caught by the `full_clean` here https://github.com/comic/grand-challenge.org/blob/122d60a8be7637df7f5b16b3520a7401b6f19e23/app/grandchallenge/cases/tasks.py#L477. However, the ValidationError that is raised there is not handled, so the entire session will fail with "An unexpected error occurred" and no images from the session will be imported. If it is handled in PanImg validation (either by truncation + warning or by rejection of that image) then the rest of the upload session is ok.

Reply:
I added it to the PR as explained in this comment: https://github.com/DIAGNijmegen/rse-panimg/pull/51#issuecomment-930114858.